### PR TITLE
add import from zuri ui to the index.ejs file

### PR DESCRIPTION
**Pull request by tundealabi**
update import path from zuri ui

### CODE REVIEW

**Review by mukhtarB — APPROVED** [review]

**Comment by mukhtarB** [comment]
@abisalde pls co-review this

**Comment by abisalde** [comment]
Looks Good ..


Can be merged

**Comment by zxenonx** [comment]
> Looks Good ..
> 
> Can be merged

@abisalde, please you should use Github options to review and approve if necessary. Actually, this PR is still waiting for your review.

**Pull request merged**
